media_driver/linux/common/ddi/media_libva_caps.cpp

```diff
@@ -1988,6 +1988,63 @@ VAStatus MediaLibvaCaps::QuerySurfaceAttributes(
             i++;
         }
     }
+    else if(entrypoint == VAEntrypointEncSlice)
+    {
+        attribs[i].type = VASurfaceAttribPixelFormat;
+        attribs[i].value.type = VAGenericValueTypeInteger;
+        attribs[i].flags = VA_SURFACE_ATTRIB_GETTABLE | VA_SURFACE_ATTRIB_SETTABLE;
+        attribs[i].value.value.i = VA_FOURCC('N', 'V', '1', '2');
+        i++;
+        
+        attribs[i].type = VASurfaceAttribMaxWidth;
+        attribs[i].value.type = VAGenericValueTypeInteger;
+        attribs[i].flags = VA_SURFACE_ATTRIB_GETTABLE | VA_SURFACE_ATTRIB_SETTABLE;
```

Comment:
The Max/Min attributes should only be GETTABLE, not SETTABLE - it doesn't make sense to pass them to vaCreateSurfaces().

Reply:
Agree

Reply:
will change it.